### PR TITLE
Adjust admonition block for dark mode (Fixes: #1522)

### PR DESCRIPTION
**Pull request by milmazz**

### CODE REVIEW

**Comment by milmazz** [comment]
## Dark

Screen Shot 2022-02-23 at 13 29 38

## Light 
Screen Shot 2022-02-23 at 13 30 07

**Comment by josevalim** [comment]
@milmazz here are the light themes, you can pick up the green one:

image

Btw, what do you think about moving those styles to their own general/admonition.css file?

File: assets/less/content/general.less [hunk 1]
```diff
@@ -188,6 +195,15 @@ blockquote {
         content: @icon-double-quotes-l;
       }
     }
+
+    &.tip {
+      background-color: #1f4034;
+
+      &::before {
+        content: @icon-error-warning;
```
Comment (josevalim) [inline assets/less/content/general.less:203]:
```suggestion
        content: @icon-information;
```

?

**Comment by milmazz** [comment]
> what do you think about moving those styles to their own general/admonition.css file?

I think that's a fantastic idea, working on it.

**Comment by milmazz** [comment]
Screen Shot 2022-02-23 at 14 05 10
Screen Shot 2022-02-23 at 14 05 23

**Review by josevalim — APPROVED** [review]

**Pull request merged**